src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -306,4 +310,77 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
     awaitCond(runAndWait(workflowQuery.findWorkflowByIds(workflowRecs.map(_.id)).map(_.status).result).forall(_ == WorkflowStatuses.Failed.toString), 10 seconds)
     workflowSubmissionActor ! PoisonPill
   }
+
+  it should "not truncate array inputs" in withDefaultTestDatabase {
+
+    mockServer.stopServer
```

Comment:
Had to do this to make the tests work properly in sbt -- there's something weird going on with the mockserver where other tests influence what happens with this.